src/stamp/preprocessing/extractor/uni2.py

```diff
@@ -0,0 +1,40 @@
+try:
+    import timm
+    from timm.data import resolve_data_config
+    from timm.data.transforms_factory import create_transform
+    import torch
+except ModuleNotFoundError as e:
+    raise ModuleNotFoundError(
+        "uni2 dependencies not installed."
+        " Please reinstall stamp using `pip install 'stamp[uni2]'`"
+    ) from e
```

Comment:
Please add these dependencies as a dependency group in the toml file: https://github.com/EzicStar/STAMP/blob/0b4760cdecd71e04cad5667d450ccbab70bbbec2/pyproject.toml#L47-L72